File: app_pojavlauncher/src/main/java/net/kdt/pojavlaunch/fragments/FabricInstallFragment.java
package net.kdt.pojavlaunch.fragments;

import net.kdt.pojavlaunch.modloaders.FabriclikeUtils;

public class FabricInstallFragment extends FabriclikeInstallFragment {

    public static final String TAG = "FabricInstallFragment";

    public FabricInstallFragment() {
        super(FabriclikeUtils.FABRIC_UTILS, TAG);
    }
}

File: app_pojavlauncher/src/main/java/net/kdt/pojavlaunch/fragments/FabricInstallFragment.kt
package net.kdt.pojavlaunch.fragments

import net.kdt.pojavlaunch.modloaders.FabriclikeUtils
import net.kdt.pojavlaunch.modloaders.ModloaderListenerProxy

class FabricInstallFragment : FabriclikeInstallFragment(FabriclikeUtils.FABRIC_UTILS, TAG) {
    companion object {
        const val TAG = "FabricInstallFragment"
    }
}
